Checkbox Example (Two State): Activate checkbox only on keyup

sivakusayan opened the pull request:
Fixes #2425.

I chose to move this logic to a `keyup` handler to maintain consistency with HTML checkboxes. If this fix is okay, we should probably make a similar fix to the [Mixed Checkbox example](https://www.w3.org/WAI/ARIA/apg/example-index/checkbox/checkbox-mixed.html).

mcking65 commented:
@howard-e @alflennik 

Note the failure of workflow
Trigger PR update for WAI-APG site / create-pr-wai

jongund reviewed (approved):
I tested the changes and it works great.
I think "spacebar" in the documentation needs to change to "space" to pass the spelling rules

sivakusayan commented:
Fixed the spell check 🙂 
I also removed the `event.preventDefault()` call in the new `keyup` handler since I realized it's no longer needed. I don't think browsers do anything when the spacebar is released — I'm guessing it was there to prevent the scrolling issue when it was previously a `keydown` handler.
